amundsen_application/api/utils/notification_utils.py

```diff
@@ -86,8 +86,8 @@ def get_notification_html(*, notification_type: str, options: Dict, sender: str)
     validate_options(options=options)
 
     url_base = app.config['FRONTEND_BASE']
-    resource_url = '{url_base}{resource_path}'.format(resource_path=options.get('resource_path'),
-                                                      url_base=url_base)
+    resource_url = '{url_base}{resource_path}?source=notification'.format(resource_path=options.get('resource_path'),
+                                                                          url_base=url_base)
```

Comment:
Not a huge deal but the code for table details only cleans up the URL if a `searchIndex` is present. This should be updated to `searchIndex !== undefined || source !== undefined`. The logging itself should still work as expected.
```
if (searchIndex !== undefined) {
  window.history.replaceState({}, '', `${window.location.origin}${window.location.pathname}`);
}
```